src/MultiCascader/MultiCascader.tsx

```diff
@@ -254,7 +254,7 @@ const MultiCascader: PickerComponent<MultiCascaderProps> = React.forwardRef(
               node.loading = false;
               node[childrenKey] = data;
 
-              if (targetRef.current) {
+              if (targetRef.current || inline) {
```

Comment:
React Hook useCallback has a missing dependency: 'inline'. Either include it or remove the dependency array